hw/ip/csrng/data/csrng.hjson

```diff
@@ -415,8 +415,6 @@
       desc: "Hardware detection of error conditions status register",
       swaccess: "ro",
       hwaccess: "hwo",
-      tags: [ // The internal HW can modify the error code registers
-              "excl:CsrAllTests:CsrExclCheck"],
```

Comment:
do we need an exclusion on enable? just not sure if the random tests can manipulate the registers in a way to accidentally cause an error.

Reply:
yes - there already is an exclusion on the master enable, as well the the ERR_CODE_TEST register. These are required to keep the ERR_CODE register "quiet". Directed tests have been written to directly test this register.